src/Storage/Controllers/ContentOnDemandController.cs

```diff
@@ -204,10 +203,15 @@ public async Task<Stream> GetFormdataAsHtml([FromRoute]string org, [FromRoute] s
         [HttpGet("formsummaryhtml")]
         public async Task<Stream> GetFormSummaryAsHtml([FromRoute] string org, [FromRoute] string app, [FromRoute] Guid instanceGuid, [FromRoute] Guid dataGuid, [FromRoute] string language)
         {
-            return await GetFormdataAsHtmlInternal(org, app, instanceGuid, dataGuid, language, 2);
+            return await GetFormdataAsHtmlStream(app, instanceGuid, dataGuid, language, 2);
         }
 
-        private async Task<Stream> GetFormdataAsHtmlInternal(string org, string app, Guid instanceGuid, Guid dataGuid, string language, int viewType, int singlePageNr = -1)
+        private async Task<Stream> GetFormdataAsHtmlStream(string app, Guid instanceGuid, Guid dataGuid, string language, int viewType, int singlePageNr = -1)
+        {
+            return new MemoryStream(Encoding.UTF8.GetBytes(await GetFormdataAsHtmlString(app, instanceGuid, dataGuid, language, viewType, singlePageNr)));
```

Comment:
I'm surprised it's not reversed. That the base method produce a stream and not a string, and then to get a string you read the stream.

Reply:
The base method is cut and paste from a2. All the logic is based on a string. If this method should produce a stream, we would introduce a back and forth conversion for the most used use case.